src/fn/fn-popover.scss

```diff
@@ -0,0 +1,127 @@
+@import "./fn-settings";
+
+$block: #{$fn-namespace}-popover;
+
+$fn-popover-arrow-size: 0.75rem;
+$fn-popover-arrow-offset: -0.25rem;
+$fn-popover-arrow-margin: 1.5rem;
+$fn-popover-arrow-middle-calc: calc(50% - 0.25rem);
+
+.#{$block} {
+  @include fn-reset();
+  @include fn-flex(column);
+
+  background-color: $fn-color-white;
+  border-radius: 0.75rem;
+  padding: 1rem 1.5rem;
+  position: relative;
+  filter: drop-shadow(0 0 0.125rem rgba(0, 0, 0, 0.25)) drop-shadow(0 0.5rem 1rem rgba(0, 0, 0, 0.25));
+
+  &__title {
+    @include fn-reset();
+
+    font-size: 1.125rem;
+    font-weight: bold;
+    line-height: 1.375rem;
+  }
+
+  &__body {
+    @include fn-reset();
+
+    margin: auto;
+    padding: 0.5rem 0 0.5rem;
+  }
+
+  &__button-container {
+    @include fn-reset();
+    @include fn-flex();
+    @include fn-set-margin-left(auto);
+
+    margin-top: auto;
```

Comment:
I thought this would be helpful in the event of a fixed-height container but have since learned that they should always fit their content so I will remove this

Reply:
After removing `margin: auto` from `fn-popover__content` I want to keep this to force the button container to the bottom